public/app/plugins/panel/flamegraph/suggestions.ts

```diff
@@ -0,0 +1,37 @@
+import { VisualizationSuggestionsBuilder } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { SuggestionName } from 'app/types/suggestions';
+
+import { FlameGraphDataContainer as FlameGraphDataContainer } from './components/FlameGraph/dataTransform';
+import { FlameGraphDataContainer as FlameGraphDataContainerV2 } from './flamegraphV2/components/FlameGraph/dataTransform';
+
+export class FlameGraphSuggestionsSupplier {
+  getListWithDefaults(builder: VisualizationSuggestionsBuilder) {
+    return builder.getListAppender<any, any>({
```

Comment:
I think this can be `builder.getListAppender<{}, {}>` that way we don't have to use `any` (which makes this less type safe) and in reality we really don't have any options for the panel right now.